08.ls_object/lib/ls/formatter.rb

```diff
@@ -4,7 +4,7 @@
 require_relative 'file'
 
 module Ls
-  class OutputFormat
+  class Formatter
     COLUMN = 3
```

Comment:
```suggestion
    COLUMN_COUNT = 3
```

のように「中身は数値ですよ」ということがすぐわかるような名前の方がベターかも